ci/kubetest/test_redis_external_with_existing_secret.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import pytest
+
+from utils import (
+    cleanup_k8s,
+    create_namespace_if_not_exists,
+    helm_install,
+    install_custom_resources,
+    is_posthog_healthy,
+    wait_for_pods_to_be_ready,
+)
+
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger()
+
+HELM_INSTALL_CMD = """
+helm upgrade \
+    --install \
+    --timeout 30m \
+    -f ../../ci/values/kubetest/test_redis_external_with_existing_secret.yaml \
+    --create-namespace \
+    --namespace posthog \
+    posthog ../../charts/posthog \
+    --wait-for-jobs \
+    --wait
+"""
+
+
+@pytest.fixture
+def setup(kube):
+    cleanup_k8s()
+    create_namespace_if_not_exists()
+    install_custom_resources("./custom_k8s_resources/redis_external_with_existing_secret.yaml")
+    helm_install(HELM_INSTALL_CMD)
+    wait_for_pods_to_be_ready(kube)
+
+
+def test_helm_install(setup, kube):
+    pass
+
+
+def test_posthog_healthy(kube):
+    is_posthog_healthy(kube)
+
+
+def test_redis_secret(kube):
+    secrets = kube.get_secrets(namespace="posthog", fields={"type": "Opaque"})
+
+    default_redis_secret_name = "posthog-posthog-redis-external"
```

Comment:
non-blocking

Seems like this might be a bit brittle e.g. we change he default name, the test doesn't fail. Could we instead validate that the existing secret is used? Fine if not.

Reply:
If you agree I'm going to fix this separately so we can land this in the meantime.